docs/integrations/sources/jira.md

```diff
@@ -126,6 +126,15 @@ This connector outputs the following incremental streams:
 
 If there are more endpoints you'd like Airbyte to support, please [create an issue.](https://github.com/airbytehq/airbyte/issues/new/choose)
 
+### Streams on I/O Usage
+
+In the list above, there is a subset of streams which requires to make one HTTP request per issue. Those streams can significantly slow down that a sync given a high number of issues. If you have one or many of those streams and experience slowness, we recommend filtering the list of issues using the list of projects in the configuration or simply removing those streams from the sync.
+* Issue properties
+* Issue remote links
+* Issue transactions
+* Issue votes
+* Issue watchers
```

Comment:
What about `issue_comments` and `issue_worklogs`?

Reply:
I did filter on substreams on `issue_property_key`. You are right that `issue_comments` and `issue_worklogs` are also at risk. I'll add those. Thanks @tolik0 !